internal/controller/nats/provisioner_test.go

```diff
@@ -50,6 +51,9 @@ func Test_handleNATSState(t *testing.T) {
 					Message:            "",
 				},
 			},
+			wantK8sEvents: []string{
+				"Normal Deploying NATS is being deployed, waiting for StatefulSet to get ready.",
```

Comment:
is the word `Normal` or `Warning` added by default to the event string?

Reply:
Yes exactly, `Normal` or `Warning` comes from the EventType